packages/headless/src/features/result-preview/result-preview-slice.test.ts

```diff
@@ -73,6 +73,18 @@ describe('ResultPreview', () => {
   });
 
   describe('on #fetchMoreResults.fulfilled', () => {
+    it('re-initialize the state to initial when a query returns correctly', () => {
```

Comment:
is there a test for the reverse condition? where the state isn't re-initialized or when a query doesn't return correctly?

Reply:
No, but IMHO, this is normal: We're testing the execution of the reducer associated with the `fetchMoreResults` actions, so testing the case where a query doesn't return correctly or 'fetchMoreResults' doesn't occur is a bit like having a function, and saying that to unit test it you need test the case where you don't call it, I found that a bit odd, because that means you don't have any 'Act' in an Arrange/Act/Assert test.